src/Protocol/ActivityPub/Processor.php

```diff
@@ -172,6 +173,38 @@ public static function dislikeItem($activity)
 		self::postItem($activity, $item);
 	}
 
+	/**
+	 * Create an event
+	 *
+	 * @param array  $activity Activity array
```

Comment:
the `$item` parameter is missing in the doxygen header

Reply:
Sorry.